akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -146,3 +146,8 @@ If you've migrated from classic remoting to Artery
 which has a completely different protocol, a rolling update is not supported.
 For more details on this migration
 see @ref:[the migration guide](../project/migration-guide-2.5.x-2.6.x.md#migrating-from-classic-remoting-to-artery).
+
+### Migrating from Classic Sharding to Typed Sharding
```

Comment:
this is a good place. I don't think I had more intentions then that it should be mentioned somewhere